eahub/localgroups/models.py

```diff
@@ -87,6 +87,15 @@ def get_local_group_types(self):
         else:
             return "Other"
 
+    def convert_to_row(self, field_names):
+        values = []
+        for field in field_names:
+            if field == "local_group_types":
+                values.append(self.get_local_group_types())
```

Comment:
I can't see 'Organisers' column in the exported csv file. Should it be there? I'm wondering if maybe it's because it's a many to many field in the model so it needs a similar function to return values as a list like has been done for local_group_types?

Reply:
Good catch! I missed it because it's a many-to-many field, which "meta.fields" does not retrieve. 
Fixed.